Apps/Src/Tasks.c

```diff
@@ -63,6 +63,8 @@ OS_MUTEX WDog_Mutex;
  */
 cell_asic Minions[NUM_MINIONS];
 uint32_t WDog_BitMap = 0;
+uint32_t Fault_BitMap = 0; //This is a variable that sets certain bits based on what caused the fault
```

Comment:
This is fine, but since we stop monitoring after we detect the first fault, we will only ever set one bit

Reply:
So are you saying that it doesn't have to be a bitmap but just a normal variable? If needed I can change it, but it seems like either way would work.

Reply:
Yeah, that's what I'm saying. I don't think we need to change it though